### PR TITLE
Include link to mosquitto documentation

### PR DESCRIPTION
**Pull request by seanvictory**
Fixes formatting and includes link to mosquitto documentation

### CODE REVIEW

File: mosquitto/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ MQTT broker for Home Assistant.
 
 ## About
 
-You can use this add-on to install Eclipse Mosquitto, which is an open-source (EPL/EDL licensed) message broker that implements the MQTT protocol. Mosquitto is lightweight and is suitable for use on all devices from low power single board computers to full servers. For more information, please see [mosquitto].
+You can use this add-on to install Eclipse Mosquitto, which is an open-source (EPL/EDL licensed) message broker that implements the MQTT protocol. Mosquitto is lightweight and is suitable for use on all devices from low power single board computers to full servers. For more information, please see [mosquitto](https://mosquitto.org/).
 
 [aarch64-shield]: https://img.shields.io/badge/aarch64-yes-green.svg
 [amd64-shield]: https://img.shields.io/badge/amd64-yes-green.svg
```
Comment (frenck) [inline mosquitto/README.md:12]:
```suggestion
You can use this add-on to install Eclipse Mosquitto, which is an open-source (EPL/EDL licensed) message broker that implements the MQTT protocol. Mosquitto is lightweight and is suitable for use on all devices from low power single board computers to full servers. For more information, please see [mosquitto].

[mosquito]: https://mosquitto.org
[aarch64-shield]: https://img.shields.io/badge/aarch64-yes-green.svg
[amd64-shield]: https://img.shields.io/badge/amd64-yes-green.svg
```

**Review by frenck — CHANGES_REQUESTED** [review]
Small tweak

**Review by frenck — APPROVED** [review]
Thanks! 👍

File: mosquitto/README.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ MQTT broker for Home Assistant.
 
 You can use this add-on to install Eclipse Mosquitto, which is an open-source (EPL/EDL licensed) message broker that implements the MQTT protocol. Mosquitto is lightweight and is suitable for use on all devices from low power single board computers to full servers. For more information, please see [mosquitto].
 
+[mosquito]: https://mosquitto.org
```
Comment (MartinHjelmare) [inline mosquitto/README.md:11]: Typo: -> `mosquitto`

**Pull request merged**